### ISSUE TITLE
Docker Images for dotifi?

### ISSUE THREAD

**Issue by ottobackwards** [palindromicity/dotifi#73]
Would it be useful to have docker images so that you could run dotifi in a docker image?  If you have trouble getting graphviz working or something?

The image should be built to be as small as possible.

**Comment by dougcooper** [comment]
That would be pretty fantastic IMO

**Comment by ChrisSamo632** [comment]
Yeah, we run things in k8s, so need to bake stuff into images. A dotifi base image would be useful for that kind of use too.

**Comment by ottobackwards** [comment]
It is in 1.0.0 project https://github.com/palindromicity/dotifi/projects/3
